maven-surefire-plugin/src/site/apt/examples/toolchains.apt.vm

```diff
@@ -54,3 +54,36 @@ Using Maven Toolchains with ${thisPlugin}.
 +---+
 
          The above example assumes that your toolchains.xml contains a valid entry with these values.
+
+* Minimal java version for toolchain
+
+    Unfortunately, the minimum version of Java that can be used as a toolchain is limited by the Surefire Booter requirement.
+
+*---------------------------------------------+--------------------------+--------------------------+
+|| <<Surefire version>>                       || <<Minimal Java version>> || <<Class File Format>>  |
+*---------------------------------------------+--------------------------+--------------------------+
+| 3.0.0-M6 or higher                          |    Java 8                |    52                    |     
+*---------------------------------------------+--------------------------+--------------------------+
+| 3.0.0-M5 and lower                          |    Java 7                |    51                    |  
+*---------------------------------------------+--------------------------+--------------------------+
+| To Be Completed                             |    Java 6                |    50                    |    
```

Comment:
Could you check Git repo for this?

Reply:
I tried but sorry this is really too time consuming.

Note that following : https://maven.apache.org/guides/mini/guide-using-toolchains.html, **Surefire** starts to support toolchain since 2.5, so theoretically no need to search before for this documentation section.

I tried to look what version was supported in 2.5 and not sure but I understand it starts with java 1.3 (see https://github.com/apache/maven-surefire/blob/surefire-2.5/pom.xml#L145-L152)

In also let a link to easily find Class File Version from Java Version : https://docs.oracle.com/javase/specs/jvms/se16/html/jvms-4.html